src/js/themes/hpe.js

```diff
@@ -103,17 +103,37 @@ export const hpe = deepFreeze({
     },
     font: {
       family: "'Metric', Arial, sans-serif",
+      // This face configuration is intended for typography exploration. Proper config
+      // will need enhancements to Grommet.
       face: `
+        @font-face {
+          font-family: "Metric";
+          src: url("https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/MetricHPEXSLight-Regular.woff2") format('woff2'),
```

Comment:
these seem to be duplicating the URLs for the `300` weight.

Reply:
^^ I believe this is also related to: https://github.com/grommet/grommet-theme-hpe/pull/291#discussion_r1026642955

Reply:
Same as the previous comment:

Yes, this is a temporary hack for preview purposes only.

The typography direction calls for Metric-light to be the new default. However, Grommet Text does not currently allow weight to be defined by the theme (only family).

For expediency, I'm proposing we use this configuration for assessing the typography styling impacts and cataloging the needed work to support.